browser/brave_wallet/brave_wallet_importer_delegate_impl.cc

```diff
@@ -270,6 +283,40 @@ void BraveWalletImporterDelegateImpl::OnGetLocalStorage(
           std::move(callback)));
 }
 
+bool BraveWalletImporterDelegateImpl::IsLegacyBraveCryptoWallet() const {
+  EthereumRemoteClientService* service =
+      EthereumRemoteClientServiceFactory::GetInstance()->GetForContext(
+          context_);
+  DCHECK(service);
+  // service->IsCryptoWalletsSetup is only for legacy 24 words mnemonic wallet
+  return service->IsCryptoWalletsSetup();
+}
+
+bool BraveWalletImporterDelegateImpl::IsBraveCryptoWalletInstalledInternal()
```

Comment:
nit: `IsCryptoWalletsInstalledInternal`

Reply:
fixed in 615ec530a09034306bb5efdea47492e0464e5558